test/images/Makefile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REGISTRY ?= gcr.io/google_containers
```

Comment:
I'd rather not add any more images to google_containers, especially not test images.

maybe we could use gcr.io/k8s-testimages? right now it's largely test-infra images, but it feels like a better fit. what do we think, @kubernetes/sig-testing-misc ?